### DISCUSSION TITLE
Industrial users text

### DISCUSSION THREAD

**Discussion by cuihtlauac** [ocaml/ocaml.org#1288] [General]
The texts stored in `data/industrial_users`, which used to be displayed in V2:

https://v2.ocaml.org/learn/companies.html

It doesn't seem to be displayed anywhere any longer. See for instance:

https://www.google.com/search?q="Citrix uses OCaml in XenServer, a world-class server virtualization system"+site:ocaml.org

Should we bring them back?

**Comment by gpetiot** [comment]
I'm in favor of bringing the description back.

I also have in mind a few more fields that would be useful:
- domain (services, fintech, healthcare, etc.)
- social media accounts (twitter, github organization, etc.)
- social media accounts of employees (twitter, github) (opt-in of course)
- what are they using OCaml for? / what are they cooking with OCaml? (github links)
